meerkat_abacus/data_management.py

```diff
@@ -873,7 +886,7 @@ def new_data_to_codes(engine=None, debug_enabled=True, restrict_uuids=None,
     country_config = param_config.country_config
 
     if restrict_uuids is not None:
-        if restrict_uuids == []:
+        if restrict_uuids == {} or sum([len(v) for v in restrict_uuids.values()]) == 0:
```

Comment:
Could you give this second condition `sum([len(v) for v in restrict_uuids.values()]) == 0` a name in form of local variable?